scripts/generate_docs.sh

```diff
@@ -51,13 +51,20 @@ fi
 if ! command -v jazzy > /dev/null; then
   gem install jazzy --no-ri --no-rdoc
 fi
+
+if [[ "$version" == "0.0.0" ]]; then
```

Comment:
`0.0.0` as a default seems kind of confusing to me. What happens if/when we tag a real 0.0.0? Maybe we should change the default to `"master"` if `git describe --abbrev=0 --tags` returns nothing.

Reply:
y i think that would be a better way to fix this

Reply:
@tanner0101 @tomerd Thanks, done! Also realised that by using `master` instead of `0.0.0` we can still use the old url prefix! 🙂